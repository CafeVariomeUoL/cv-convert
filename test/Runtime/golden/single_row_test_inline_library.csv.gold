[
{
    "AttM_MMSE-V2": "999",
    "Met_Date_V1_DoctorId": "1",
    "Met_Date_V2_Doctorid": "999",
    "AttM_Right_Hippocampus_mm3": "5197.9",
    "hello": "world",
    "Att_APOE": "Apo-ε3/ε3",
    "Prop_Date_V2": "999",
    "Prop_Date_Visita_MRI": "15/07/2014",
    "Met_Date_V2_LengthofTest": "999",
    "Met_Date_V1_LengthofTest": "24",
    "AttM_Total_Intracraneal_Volume_mm3": "1544503.965",
    "Prop_Date_V1": "15/10/2013",
    "Att_Years_of_education": "18",
    "AttM_Left_Hippocampus_TIV": "0.004284241",
    "AttM_Left_Hippocampal_MTA": "Score 1:  Sólo la ampliación de la fisura coroide",
    "AttM_Brain_seg_mm3": "1168165",
    "AttM_MMSE-V1": "30",
    "AttM_Right_Hippocampus_TIV": "0.004449628",
    "Att_AD_family_History": "mother",
    "AttM_Right_Hippocampal_MTA": "Score 1:  Sólo la ampliación de la fisura coroide",
    "AttM_Left_Hippocampapus_mm3": "5004.7",
    "AttM_MIS-V2": "999",
    "Att_Diagnosis": "mci",
    "subject_id": "1",
    "AttM_MIS-V1": "8",
    "Att_Gender": "Male",
    "i": 0
}
]