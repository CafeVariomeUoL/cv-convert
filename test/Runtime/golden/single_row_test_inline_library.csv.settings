{
  "processFunction": "return Lib.helloWorld(row);",
  "libraryFunctions": "Lib.helloWorld = row => {return {...row, hello: \"world\"}};"
}