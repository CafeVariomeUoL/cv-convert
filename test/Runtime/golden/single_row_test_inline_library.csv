subject_id,Att_Diagnosis,Att_Gender,Att_Years_of_education,Att_APOE,Att_AD_family_History,AttM_MMSE-V1,AttM_MIS-V1,Prop_Date_V1,Met_Date_V1_LengthofTest,Met_Date_V1_DoctorId,AttM_MMSE-V2,AttM_MIS-V2,Prop_Date_V2,Met_Date_V2_LengthofTest,Met_Date_V2_Doctorid,AttM_Total_Intracraneal_Volume_mm3,AttM_Brain_seg_mm3,AttM_Left_Hippocampapus_mm3,AttM_Right_Hippocampus_mm3,AttM_Left_Hippocampus_TIV,AttM_Right_Hippocampus_TIV,AttM_Left_Hippocampal_MTA,AttM_Right_Hippocampal_MTA,Prop_Date_Visita_MRI
1,mci,Male,18,Apo-ε3/ε3,mother,30,8,15/10/2013,24,1,999,999,999,999,999,1544503.965,1168165,5004.7,5197.9,0.004284241,0.004449628,Score 1:  Sólo la ampliación de la fisura coroide,Score 1:  Sólo la ampliación de la fisura coroide,15/07/2014
